api/node_pool.go

```diff
@@ -21,6 +16,10 @@ type NodePool struct {
 	InstanceType string
 }
 
+func (np NodePool) IsSpot() bool {
+	return np.Profile == "spot_max_price" || np.Profile == "spot"
```

Comment:
`strings.Contains("spot")` maybe?

Reply:
This is just temporary so we could migrate to `spot`. Or do you think we'll have other spot profiles in the future?

Reply:
But yeah, we could just go with it. Fixed.

Reply:
@linki I think if the need for more profiles shows up in the future, it can be addressed in another PR

Reply:
I see, didn't know it's a temporary thing. Fine for me like this as well.